targets/TARGET_STM/TARGET_STM32H7/device/stm32h7xx_ll_usart.h

```diff
@@ -82,7 +82,7 @@ static const uint16_t USART_PRESCALER_TAB[] =
 #endif /*USE_FULL_LL_DRIVER*/
 
 /* Exported types ------------------------------------------------------------*/
-#if defined(USE_FULL_LL_DRIVER)
+#if 1 //mbed
```

Comment:
we dont rather define the macro as 1 instead of rewriting the driver files?

Reply:
You're right. I have done like this because it was already the case before...
I check how to clean this part.